applications/incompressible_navier_stokes/poiseuille/input.json

```diff
@@ -7,16 +7,19 @@
     "SpatialResolution": {
         "DegreeMin": "2",
         "DegreeMax": "2",
-        "RefineSpaceMin": "0",
-        "RefineSpaceMax": "0"
+        "RefineSpaceMin": "1",
```

Comment:
Changes to the input.json here will be removed completely - just to show how I tested.